test/recipes.spec.js

```diff
@@ -0,0 +1,167 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../server';
+
+chai.use(chaiHttp);
+
+const recipe = {
+  title: 'How to prepare your finest recipe',
+  ingredients: ['A spoon of awesomeness', 'A cup of dedication'],
+  steps: {
+    '1': {
+      description: 'Add a spoon of awesomeness to the mixer',
+      images: 'https://i.stack.imgur.com/xHWG8.jpg'
+    },
+    '2': {
+      description: 'Add a spoon of awesomeness to the mixer',
+      images: 'https://i.stack.imgur.com/xHWG8.jpg'
+    }
+  },
+  cookingTime: 1000,
+  preparationTime: 3000,
+  tagList: ['food', 'meal']
+};
+
+const token =
+  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiZW1haWwiOiJqYWtlQGpha2UuamFrZSIsImlhdCI6MTU1MjM4MTYwNiwiZXhwIjoxNTUyOTg2NDA2fQ.cjn-7GZa78CW0IrE_X1erByHi4-6LEfco3hvK6A-rjQ';
+
+describe('Create New Recipe', () => {
+  describe('POST /api/recipes', () => {
+    const baseUrl = '/api/recipes';
+    it('should post new recipe when token is provided', done => {
+      chai
+        .request(server)
+        .post(baseUrl)
+        .set({ accesstoken: token })
+        .send(recipe)
+        .end((err, res) => {
+          const { body } = res;
+          expect(res).to.have.status(201);
+          expect(body.recipe).to.be.an('object');
+          expect(body.recipe.id).to.be.a('number');
+          expect(body.recipe.title).to.be.a('string');
```

Comment:
Shouldn't we test that the attributes returned match what was sent?

Reply:
Okay. I will test that the response is same as the input.